x/govshuttle/keeper/msg_server_test.go

```diff
@@ -157,6 +174,22 @@ func (suite *KeeperTestSuite) TestMsgExecutionByProposal() {
 			func(uint64, sdk.Msg) {},
 			true,
 		},
+		{
+			"fail - MsgTreasuryProposal - validate basic logic",
```

Comment:
It seems that the reason this test case is failing is not related to validate basic. Could you please check it?

Reply:
ditto